### PR TITLE
Add Play Google auth.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Use the Play sbt plugin for Play projects
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.1")
```
Comment (mchv) [inline project/plugins.sbt]: may be directly to `2.5.4`

**Comment by mchv** [comment]
Code looks good but as usual you can't check it works without deploying it and do some manual testing!
👍  when it will compile

**Comment by adamnfish** [comment]
FYI for Play 2.5 there's a new version of the play googleauth library that's substantially easier to integrate! Only released it this week so you probably started before it was there. You might like to upgrade to use that though, since it moves the oauth callback logic out of your controller and makes it much much easier to enforce any required Google Groups.

Here's the example for Play-GoogleAuth `0.5.0`:
https://github.com/guardian/play-googleauth/blob/master/example/app/controllers/Login.scala

**Comment by LATaylor-guardian** [comment]
@adamnfish I actually done this last night quickly and hadn't even realised there was a newer version. Thanks for letting me know, will update today.

**Pull request merged**